### DISCUSSION TITLE
Parse Makefile for upgrade

### DISCUSSION THREAD

**Discussion by SamuelCabralCruz** [renovatebot/renovate#10427] [Request Help]
**What would you like Renovate to be able to do?**

I am new to golang and still learning a lot every day, but it seems to be standard to install development dependencies only using Makefile for simpler CI pipeline design. 
One such an example would be that I use gofumpt and revive to lint my code:

```
.PHONY: install
install:
	@go mod download
	@go install github.com/mgechev/revive@latest
	@go install mvdan.cc/gofumpt@latest

.PHONY: lint
lint:
	@gofumpt -l -s . | grep ".*\.go"; if [ "$$?" = "0" ]; then exit 1; fi
	@revive -config revive.toml -formatter friendly .
```

As you can see here, I used the `@latest` tag instead of pinning a version.
I know that renovate is good at scanning many different types of files on the lookout for dependencies.
One such an example is the upgrade of github actions version within the `.github/workflows` folder. 

I was wondering if it would be a good idea to look into Makefile to provide all the goodies that renovate offer such as pining versions and updating them on new releases.

**Did you already have any implementation ideas?**

I am really clueless about the way renovate works.

**Comment by viceice** [comment]
You can use the [regex](https://docs.renovatebot.com/modules/manager/regex/) manager.

https://docs.renovatebot.com/configuration-options/#regexmanagers
